digital-workplace-webapps/src/main/webapp/skin/less/digital-workplace.less

```diff
@@ -3,6 +3,11 @@
 /*--------------- HomePage Containers Structure ---------------------------*/
 #digitalWorkplaceHomePage {
 
+  #SliderContainer .newsSlider .sliderNewsItems  {
+    width: 100%;
+    height: 220px !important;
```

Comment:
if this part is related to the template itself, it would be better to put it in news addon, otherwises the template will not display correctly without DW addon